en/ui-system/components/editor/webview.md

```diff
@@ -6,48 +6,47 @@
 
 Click **Add Component** at the bottom of **Properties** panel and select **WebView** from **UI Component** to add the WebView component to the node.
 
-For more information about WebView API, please refer to [WebView API](../../../api/en/classes/WebView.html) documentation.
+For more information, please refer to the [WebView API](https://docs.cocos.com/creator/3.0/api/en/classes/WebView.html) documentation.
```

Comment:
../../../api/zh/classes/webview.webview-1.html

Reply:
I did not change this, it is incorrect in the `v3.0` branch which was the original branch I was using for this PR until you had me change it to `v3.0-release`

https://raw.githubusercontent.com/cocos-creator/creator-docs/v3.0/en/ui-system/components/editor/webview.md